testData/bugs/_110/TargetNoPropertyMethode.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at https://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.example;
+
+public class TargetNoPropertyMethode {
```

Comment:
Let's rename this to `TargetNoPropertyMethods`

Reply:
Renamed.